### PR TITLE
feat(stock): add currency to expiration report

### LINKED ISSUE #5474: Add currency to Expiration Report
We should add the ability to select a currency to render the Expiration Report in.  Here is what it look like currently:
image

To satisfy this issue, we would need to:

1. The `bhCurrencySelect` to the page.
2. Set the enterprise currency by default.
2. Add logic to convert the values on the page into the selected currency at the exchange rate of the day.
3. Add text to the report that mentions the current exchange rate, if values are not in the enterprise currency.

### PR DESCRIPTION
**Pull request by jniles**
Adds the currency to the stock expiration report so that users can see the value of the stock at the current exchange rate.

Closes #5474.

Here is what it looks like:

image

(Normally this should be in FC)

### CODE REVIEW

**Comment by mbayopanda** [comment]
On the fresh bhima_test database I got this : 
I0JW0lYBow

- The currency doesn't change
- For $ as enterprise currency, the report gives zeros while for other currencies there are not zeros values

I ran : 
- yarn build:db
- yarn dev (I tested the report in the app)

**Comment by jniles** [comment]
Thanks for the review!  Let me take a look at this and make sure it is behaving correctly.

**Comment by jniles** [comment]
This PR had multiple problems and wasn't at all ready.   Sorry!  Now all the problems should be fixed.  Could you give it another look @mbayopanda ?

image

File: server/controllers/stock/reports/stock/expiration_report.js [hunk 1]
```diff
@@ -34,6 +44,15 @@ async function stockExpirationReport(req, res, next) {
...
     }
 
+    const isEnterpriseCurrency = currencyId === req.session.enterprise.currency_id;
+
+    const [{ rate }] = await db.exec(
+      'SELECT GetExchangeRate(?, ?, NOW()) AS rate;',
+      [req.session.enterprise.id, currencyId],
+    );
+
+    const exchangeRate = isEnterpriseCurrency ? 1 : rate;
```
Comment (mbayopanda) [inline server/controllers/stock/reports/stock/expiration_report.js:54]: 👍🏽

**Review by mbayopanda — APPROVED** [review]
LGTM 👍🏽 

bors r+

**Pull request merged**